packages/ui/cypress/support/commands.ts

```diff
@@ -85,19 +85,24 @@ Cypress.Commands.add('rejectTx', (id: number, reason: string) => {
 
 Cypress.Commands.add('rejectCurrentMultisigTx', rejectCurrentMultisigTxs)
 
-Cypress.Commands.add('connectAccounts', (accountAddresses = [Account1] as string[]) => {
-  cy.getAuthRequests().then((authRequests) => {
-    const requests = Object.values(authRequests)
-    // we should have 1 connection request to the extension
-    cy.wrap(requests.length).should('eq', 1)
-    // this request should be from the application Multix
-    cy.wrap(requests[0].origin).should('eq', 'Multix')
-    // let's allow Accounts to connect
-    cy.enableAuth(requests[0].id, accountAddresses)
-    // the ui should then move on to connecting to the rpcs
-    topMenuItems.multiproxySelector().should('be.visible')
-  })
-})
+Cypress.Commands.add(
+  'connectAccounts',
+  (accountAddresses = [Account1] as string[], checkMultiProxyVisible: boolean = true) => {
```

Comment:
@Tbaut yeah it will fail without it because of the line `topMenuItems.multiproxySelector().should('be.visible')` as the multiproxy doesn't appear in two cases when

- if not enabling (sharing?) the initialized account (but need to respond to the auth request to continue)
- one of the accounts is not already part of a multisig.

 I see your point though, we are using the connectAccount function but not fully connecting it when using it like this. Perhaps the answer is a separate function.

Reply:
What I meant by
> such a test should actually not be part of the `connectAccount` funtion

Is the test to check for `topMenuItems.multiproxySelector().should('be.visible')`.
This should be removed, the check for the multiproxy. And then see if other tests fail.

Reply:
@Tbaut Ah ok, I assumed it was necessary but after just removing that check all together the rest of the tests are still passing...yeah this is simpler!